data/json/items/tools.json

```diff
@@ -2053,6 +2054,7 @@
   {
     "id": "cs_lajatang_off",
     "type": "TOOL",
+    "category": "weapon",
```

Comment:
We do not have category named `weapon`. It is called `weapons`.

![image](https://user-images.githubusercontent.com/16213433/47555027-7aa46400-d913-11e8-8172-abc5fff7cbb0.png)